sdk/storage/test/append_blob_client_test.cpp

```diff
@@ -57,19 +57,19 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_EQ(properties.ContentLength, static_cast<int64_t>(m_blobContent.size()));
 
```

Comment:
TODO: Add test condition test cases.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/261